README.md

```diff
@@ -366,6 +366,14 @@ You can create your own environment names by adding url fields to your `sites` p
 
 The `ENV` environment variable determines which environment will be used. If it is `prod` for example, the url used will be the one defined in the schema or in the sites configuration for `prod`.
 
+## Resource leak mitigation
+
+If you suspect your application is slowly leaking memory, HTTP sockets or some other resource that eventually renders it nonresponsive, you can set the `maxRequestsBeforeShutdown` option. The application will automatically exit after that number of requests. By default, this mechanism calls `process.exit(0)`. You can change this behavior by passing a custom `exit` function to `apostrophe-multisite` as an option.
```

Comment:
I think we should provide at least a loose suggestion for number of requests here, possibly referencing the PHP implementation.

Reply:
I suggested 10000.